projects/packages/my-jetpack/_inc/components/product-cards-section/index.tsx

```diff
@@ -90,17 +92,48 @@ const DisplayItems: FC< DisplayItemsProps > = ( { slugs } ) => {
 
 interface ProductCardsSectionProps {
 	noticeMessage: ReactNode;
+	siteIsRegistered?: boolean;
 }
 
-const ProductCardsSection: FC< ProductCardsSectionProps > = ( { noticeMessage } ) => {
-	const { ownedProducts = [], unownedProducts = [] } =
+const ProductCardsSection: FC< ProductCardsSectionProps > = ( {
+	noticeMessage,
+	siteIsRegistered,
+} ) => {
+	const { ownedProducts: initialOwnedProducts = [], unownedProducts: initialUnownedProducts = [] } =
 		getMyJetpackWindowInitialState( 'lifecycleStats' );
+	const [ productsOwnership, setProductsOwnership ] = useValueStore( 'productsOwnership', {
+		ownedProducts: initialOwnedProducts,
```

Comment:
If there is no real need to expose explicitly `initialOwnedProducts`, there is no issue in just referencing `getMyJetpackWindowInitialState()` function directly here. What do you think?
```suggestion
		ownedProducts: getMyJetpackWindowInitialState( 'lifecycleStats' )?.ownedProducts ?? [],
```